apps/are_form/lib/classmydates.js

```diff
@@ -66,8 +66,8 @@ var moment = require('moment');
 
 	Date.prototype.addDaysIgnoringWeekends = function(daysToAdd) {
 
-		var count = 0;
-		var myDate = new Date(this.valueOf());
+		let count = 0;
+		let myDate = new Date(this.valueOf());
 		while (count < daysToAdd) {
 			myDate.setDate(myDate.getDate() + 1);
 			if (myDate.isWeekend() == false) {
```

Comment:
do all of these have to be `let` rather than `const`

Reply:
the variables are updated in the while loops, probably needs refactoring at some point.

Reply:
keep for now too much to refactor